Paintroid/src/main/res/layout/dialog_pocketpaint_shapes.xml

```diff
@@ -17,23 +17,133 @@
+
+        <EditText
+            android:id="@+id/pocketpaint_shape_outline_edit"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="1"
+            android:layout_gravity="center"
+            android:layout_marginTop="16dp"
+            android:inputType="number"
+            android:imeOptions="actionDone"
+            android:textColor="@color/pocketpaint_main_rectangle_tool_accent_color"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            tools:ignore="LabelFor" />
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/pocketpaint_shape_tool_fill_outline"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
+        android:layout_marginTop="16dp"
+        android:text="@string/shape_tool_dialog_fill_title"
+        android:textAllCaps="true"
+        android:textColor="@color/pocketpaint_main_rectangle_tool_accent_color"
+        android:textStyle="bold"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="24dip" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="2dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:background="@color/pocketpaint_main_separator_color"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="16dp" />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_gravity="center"
+        android:orientation="horizontal" >
+
+        <ImageButton
+            android:id="@+id/pocketpaint_shape_ibtn_fill"
+            style="@style/PocketPaintSelectableButton"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:padding="16dp"
+            android:src="@drawable/ic_pocketpaint_rectangle" />
+
+        <ImageButton
+            android:id="@+id/pocketpaint_shape_ibtn_outline"
+            style="@style/PocketPaintSelectableButton"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:padding="16dp"
+            android:src="@drawable/ic_pocketpaint_rectangle_out" />
+    </LinearLayout>
     <TextView
         android:id="@+id/pocketpaint_shape_tool_dialog_title"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:text="@string/shape_tool_dialog_rect_title"
         android:textAllCaps="true"
-        android:textColor="@color/pocketpaint_colorAccent"
+        android:textColor="@color/pocketpaint_main_rectangle_tool_accent_color"
         android:textStyle="bold"
         android:layout_marginStart="24dp"
         android:layout_marginEnd="16dp"/>
 
-    <View style="@style/PocketPaintDividerHorizontal"/>
+    <View
```

Comment:
Please revert this and use the `<View style="@style/PocketPaintDividerHorizontal"/>` to avoid duplication.